benchmarks/ekomark/plots.py

```diff
@@ -71,7 +71,7 @@ def plot_dist(x, y, yerr, yref, title=None, oMx_min=1e-2, oMx_max=0.5):
     ax3 = plt.subplot(2, 3, 3)
     ax3.set_xscale("log", nonpositive="clip")
     ax3.set_yscale("log", nonpositive="clip")
-    oMx = 1.0 - x
+    oMx = [1.0 - i for i in x]
```

Comment:
Actually it should work even as it was before, if `x` is in array (and so `oMx` will be as well). If for some reason it happens not to be an array you can add somewhere:
```py
x = np.array(x)
```

Reply:
Fixed